client/src/components/views/Production.jsx

```diff
@@ -0,0 +1,10 @@
+import ProductionModal from 'components/Production/ProductionModal.jsx';
+import ProductionTable from 'components/Production/ProductionTable.jsx';
+
+const Productions = () => {
+  <ProductionModal />;
+
+  return <ProductionTable />;
```

Comment:
Can you add the `Head` component like here: https://github.com/Mackbellemore/soen-390-team07/blob/09f7d5fbf86eb06130890356dfcdb3aa868b8959/client/src/components/views/AuditTrail.jsx#L8-L11